publish2cloud.py

```diff
@@ -205,7 +206,8 @@ def publish_to_s3(config, section, chunknum):
     for key_name in (chunk_key, key):
         k = boto.s3.key.Key(bucket)
         k.key = key_name
-        k.set_contents_from_filename(output_filename, policy='bucket-owner-full-control')
+        k.set_contents_from_filename(output_filename)
+        k.set_acl('bucket-owner-full-control')
```

Comment:
```suggestion
        k.set_acl('public-read')
```
According to the documentation [here](http://boto.cloudhackers.com/en/latest/s3_tut.html#setting-getting-the-access-control-list-for-buckets-and-keys), if we want to have the files be publicly readable the policy should be `public-read`.